lib/msal-core/src/cache/AuthCache.ts

```diff
@@ -206,6 +208,31 @@ export class AuthCache extends BrowserStorage {// Singleton
         return results;
     }
 
+    /**
+     * Get idToken in the cache
+     * @param  
+     */
+    getIdToken(clientId: string, homeAccountIdentifier: string): AccessTokenValue {
+        const key = Object.keys(window[this.cacheLocation]).find((key) => {
+            const keyMatches = key.match(clientId) && key.match(homeAccountIdentifier);
```

Comment:
The idToken cacheKey should be a standard format of `msal.clientId.idtoken`

Since this is in AuthCache you can just call `this.generateCacheKey(PersistentCacheKeys.IDTOKEN, true)` to get the idtoken key